src/main/resources/application.properties

```diff
@@ -32,14 +32,14 @@ rabbitmq.user: myuser
 rabbitmq.password: myuser
 # Check configuration documentation for using jasypt encrypted passswords
 # Valid TLS versions: 'TLSv1.2', 'TLSv1.1', 'TLSv1', 'TLS', 'SSLv3', 'SSLv2', 'SSL'
-rabbitmq.tlsVersion:
+rabbitmq.tls.version:
 rabbitmq.exchange.name: ei-exchange
-rabbitmq.domainId: ei-domain
-rabbitmq.componentName: eiffel-intelligence
-rabbitmq.consumerName: messageConsumer
-rabbitmq.queue.durable:true
-rabbitmq.binding.key:#
-rabbitmq.waitlist.queue.suffix:waitlist
+rabbitmq.domain.id: ei-domain
+rabbitmq.component.name: eiffel-intelligence
+rabbitmq.consumer.name: messageConsumer
+rabbitmq.queue.durable: true
+rabbitmq.binding.key: #
+rabbitmq.wait.list.queue.suffix: waitList
```

Comment:
I don't know if we should have waitlist or wait.list .... Is it one word we've created or two?

Reply:
I think it's two words but I just put them back together for simplicity.